packages/react-instantsearch-hooks/src/useInstantSearch.ts

```diff
@@ -2,11 +2,25 @@ import instantsearch from 'instantsearch.js';
 import { useEffect, useMemo, version as ReactVersion } from 'react';
 
 import { useForceUpdate } from './useForceUpdate';
+import { useInstantSearchServerContext } from './useInstantSearchServerContext';
+import { useInstantSearchSSRContext } from './useInstantSearchSSRContext';
 import { useStableValue } from './useStableValue';
 import { warn } from './utils';
 import version from './version';
 
-import type { InstantSearchOptions } from 'instantsearch.js';
+import type { InstantSearchServerApi } from './InstantSearchServerContext';
+import type { InstantSearchServerState } from './InstantSearchSSRProvider';
+import type {
+  InstantSearchOptions,
+  InstantSearch,
+  SearchClient,
+} from 'instantsearch.js';
+
+const defaultUserAgents = [
+  `react (${ReactVersion})`,
+  `react-instantsearch (${version})`,
```

Comment:
React InstantSearch v6 added a specific server UA for the backend request. Do you think that's useful?

Reply:
The server UA is covered here: https://github.com/algolia/react-instantsearch/blob/f3b3533cd7b436fffa34f3e8748e5308d938a258/packages/react-instantsearch-hooks/src/useInstantSearch.ts#L102-L107

Did you mean there's another one? I can't find it in the codebase.

Reply:
I missed this part, thanks!